tests/functional/FileApiFunctionalTest.php

```diff
@@ -158,6 +159,45 @@ public function testFileApiDownloadFile() {
         }
     }
 
+    /**
+     * Test for download all translations of file.
+     */
+    public function testFileApiDownloadAllTranslationsOfFile() {
+        try {
+            $params = new DownloadFileParameters();
+            $params->setRetrievalType($this->retrievalType);
+            $result = $this->fileApi->downloadAllTranslationsOfFile(self::FILE_NAME, $params);
+
+            $this->assertInstanceOf(Stream::class, $result);
+        } catch (SmartlingApiException $e) {
+            $this->fail($e->getMessage());
+        }
+    }
+
+    /**
+     * Test for download multiple translations of files.
+     */
+    public function testFileApiDownloadMultipleTranslationsOfFiles() {
```

Comment:
This test will fail until [this FileAPI endpoint](https://api-reference.smartling.com/#operation/downloadMultipleTranslatedFiles) receives parameters without `[]`.